src/tests/JIT/HardwareIntrinsics/Arm/Shared/_SveBinaryOpTestTemplate.template

```diff
@@ -0,0 +1,328 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
What's "different" about this compared to the `_BinaryOpTestTemplate.template`?

I would have expected them to be essentially identical and any minor differences could likely have been adjusted, rather than requiring an entirely new template.